module/zstd/zstd.c

```diff
@@ -0,0 +1,526 @@
+/*
+ * Common functions of New Generation Entropy library
```

Comment:
The git rename detection seems to have incorrectly stuck the license and description from 'common/entropy_common.c' on this file.

Reply:
can be also my fault. i was asked to change the license from cddl to bsd. dont know what went wrong here. will fix that